core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -140,6 +140,7 @@ public CallOptions withCompressor(@Nullable Compressor compressor) {
    * TLS certificate.</em>
    */
   @Nullable
+  @ExperimentalApi
```

Comment:
Sorry it's not obvious to me why this is experimental. It would be useful if the added experimental annotations come with a few words for the reason.